src/store/actions.ts

```diff
@@ -313,15 +313,11 @@ export async function searchBusiness ({ getters }, corpNum: string): Promise<Bus
       // now try to find business in COLIN
       try {
         const data = await NamexServices.searchColin(corpNum)
-        if (data.directors === 'Not Available' && data.incorporated === 'Not Available') {
-          return Promise.reject(error) // business not found
-        }
-        // *** TODO: update this when new endpoint is available
         return {
-          identifier: corpNum, // data['incorp #'],
-          legalName: 'Future Name From COLIN',
-          legalType: EntityTypes.BC,
-          state: EntityStates.ACTIVE
+          identifier: data.business.identifier,
+          legalName: data.business.legalName,
+          legalType: data.business.legalType,
+          state: data.business.state
```

Comment:
@severinbeauvais do we require state value? it doesn't seems to be used anywhere. this api call doesn't return state (pr is still in review)

Reply:
I think we need to know if the business is active or historical ...?

Reply:
not sure if colin has the same list of state as lear (i believe active should be there)

Reply:
Does the UI need to know whether the returned business is active or historical? Or that is a search query parameter?

Reply:
I don't see its been used in the UI, probably not required

Reply:
Check with @JazzarKarim . I don't remember if the UI needs to filter for active/historical or whether that's handled by the API call (ie, only return active or historical or all).